CONTRIBUTING.md for org-wide repo (#13)
auth0/open-source-template: GENERAL-CONTRIBUTING.md

- Only fix/add the functionality in question **OR** address wide-spread whitespace/style issues, not both.
- Add unit or integration tests for fixed or changed functionality (if a test suite already exists).
- Address a single concern in the least number of changed lines as possible.
- Include documentation in the repo or on our [docs site](https://auth0.com/docs).
- Be accompanied by a complete Pull Request template (loaded automatically when a PR is created).

For changes that address core functionality or would require breaking changes (e.g. a major release), it's best to open an Issue to discuss your proposal first. This is not required but can save time creating and reviewing changes.

In general, we follow the ["fork-and-pull" Git workflow](https://github.com/susam/gitpr)

1. Fork the repository to your own Github account
2. Clone the project to your machine
3. Create a branch locally with a succinct but descriptive name
4. Commit changes to the branch
5. Following any formatting and testing guidelines specific to this repo
6. Push changes to your fork
7. Open a PR in our repository and follow the PR template so that we can efficiently review the changes.

> 

Reviewer: I know we already have this in the issue templates, but I wonder if it makes sense to keep some wording about responsible vulnerability disclosure in the code of conduct because it is so important. I am biased though :)

Author: I considered that ... I left it out because it's very clear in the Issue templates, didn't seem worth it to repeat that in this context (when you might be exploring but nothing specific in mind). I don't feel strongly, though, so I'll add it back in if you think it would filter more of that kind of request out.

Reviewer: I wonder if legal has an opinion on this. My security hat says leave it in, perhaps with some modified wording and make it an expectation for the community to be responsible with disclosing vulnerabilities.

Author: Added a bullet under Getting Started

A big welcome and thank you for considering contributing to Auth0 open source projects! It’s people like you that make it a reality for users in our community.

Reading and following these guidelines will help us make the contribution process easy and effective for everyone involved. It also communicates that you agree to respect the time of the developers managing and developing these open source projects. In return, we will reciprocate that respect by addressing your issue, assessing changes, and helping you finalize your pull requests.

## Quicklinks

* [Code of Conduct](#code-of-conduct)
* [Getting Started](#getting-started)
    * [Issues](#issues)
    * [Pull Requests](#pull-requests)
* [Getting Help](#getting-help)

## Code of Conduct

We take our open source community seriously and hold ourselves and other contributors to high standards of communication. By participating and contributing to this project, you agree to uphold our [Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

## Getting Started

Contributions are made to this repo via Issues and Pull Requests (PRs). A few general guidelines that cover both:

> - To report security vulnerabilities, please follow [Responsible Disclosure Program](https://auth0.com/whitehat) details the procedure for disclosing security issues instead of creating an Issue or PR in this repo.

Reviewer: ```suggestion
- To report security vulnerabilities, please use our [Responsible Disclosure Program](https://auth0.com/whitehat) which is monitored by our security team.
```
